packages/calcite-components/src/components/slider/slider.scss

```diff
@@ -133,55 +162,66 @@
 
   .handle-extension {
     @apply w-0.5;
-    block-size: var(--calcite-slider-handle-extension-height);
-    background-color: var(--calcite-color-text-3);
+    block-size: var(--calcite-internal-slider-handle-extension-height);
+    background-color: var(--calcite-slider-handle-extension-color, var(--calcite-color-text-3));
   }
 
-  &:hover {
+  &:focus {
+    .handle {
+      @apply focus-outset;
+    }
+  }
+
+  &--active,
+  &:focus {
     .handle {
-      box-shadow: 0 0 0 3px var(--calcite-color-brand) inset;
+      background-color: var(--calcite-slider-handle-color-active, var(--calcite-color-brand));
+      box-shadow: 0 0 0 3px var(--calcite-slider-handle-border-color-hover, var(--calcite-color-brand)) inset;
```

Comment:
should this be `box-shadow: var(--calcite-slider-handle-shadow, 0 0 0 3px var(--calcite-slider-handle-border-color-hover, var(--calcite-color-brand)) inset);` ?
this applies to other box-shadows on the handle

Reply:
I had considered something like that, but to a consumer, the inset box-shadow is used as a border. From that perspective, I think it'd be better to only expose color props for the handle's "border". This might affect other similar shadow props. I know I'll have to update `switch` to follow suit. cc @ashetland @macandcheese